### PR TITLE
New package: logiops-0.2.3

### PR DESCRIPTION
**Pull request by T0mstone**
#### New package
- This new package conforms to the [package requirements](https://github.com/void-linux/void-packages/blob/master/CONTRIBUTING.md#package-requirements): **YES**

#### Local build testing
- I built this PR locally for my native architecture, (x86_64-glibc)

### CODE REVIEW

File: srcpkgs/logiops/files/logid.cfg [hunk 1]
```diff
@@ -0,0 +1 @@
+devices: ();
```
Comment (Duncaen) [inline srcpkgs/logiops/files/logid.cfg:1]: Its easier to simply not ship a configuration file and let the user create it.

File: srcpkgs/logiops/files/logid/run [hunk 2]
```diff
@@ -0,0 +1,7 @@
+#!/bin/sh
+if /usr/bin/modprobe hid_logitech_hidpp
+then exec /usr/bin/logid
+else
+	# retry every second until the module is loaded
+	sleep 1
+fi
```
Comment (Duncaen) [inline srcpkgs/logiops/files/logid/run:7]: The file should have a trailing newline and probably look like
```
#!/bin/sh
[ -r conf ] && . ./conf
modprobe -q hid_logitech_hidpp || exit 1
exec logid ${OPTS}

**Review by Duncaen — CHANGES_REQUESTED** [review]
see comments.

**Comment by T0mstone** [comment]
Thanks for the feedback! Is it fine now? (I'll squash the branch if you want me to – I've kept it as a separate commit for now to keep the PR history)

**Pull request closed without merging**